content/source/docs/enterprise/api/plan-exports.html.md

```diff
@@ -0,0 +1,215 @@
+---
+layout: enterprise2
+page_title: "Plan Exports - API Docs - Terraform Enterprise"
+sidebar_current: "docs-enterprise2-api-plan-exports"
+---
+
+# Plans API
+
+-> **Note**: These API endpoints are in beta and are subject to change.
+
+Plan exports allow users to download data exported from the executable plan of a Run in a Terraform workspace. Currently, the only supported format for exporting plan data is to generate mock data for Sentinel. We may support additional formats in the future.
+
+## Create a plan export
+
+`POST /plan-exports`
+
+This endpoint exports data from a plan in the specified format. The export process is asynchronous, and the resulting data becomes downloadable when its status is `"finished"`. The data is then available for one hour before expiring. After the hour is up, a new export can be created.
+
+Status  | Response                                       | Reason
+--------|------------------------------------------------|----------
+[201][] | [JSON API document][] (`type: "plan-exports"`) | Successfully created a policy
+[404][] | [JSON API error object][]                      | Plan not found, or user unauthorized to perform action
+[422][] | [JSON API error object][]                      | Malformed request body (missing attributes, wrong types, etc.)
```

Comment:
What happens if there is an existing export?

Reply:
That's a 422 as well. I can specify that